samples/boards/x86_info/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2019 Intel Corp.
+
+cmake_minimum_required(VERSION 3.8.2)
```

Comment:
all other places is setting 3.13.1. Don't know cmake enough to know if this version will be checked later or if it will explode when trying to use something from 3.13

Reply:
Oh, wow. I'll fix this and file a bug - we have at handful of other tests/samples in the tree that set the wrong minimum (evidently I copied the boilerplate from one of those!).  

I seem to recall some weird thing with the way we use cmake that the minimum verison is ONLY enforced when it's in the top-level (in our case, app) `CMakeLists.txt`..

Reply:
fixed!